docs/api-and-integration/metadata/jira.md

```diff
@@ -1,55 +1,91 @@
-    - Release Note Prefix: Specifying a 'release note prefix' tells Octopus to search through your Jira issue description for that value. If it finds a match, then it will use the issue description as the release note; otherwise, it will use the issue title. See [Release Notes](/docs/api-and-integration/metadata/release-notes-templates.md) for more information.
+    - **Jira Base URL**. This tells Octopus where your Jira instance is located and enables Octopus to render the links back to Jira issues. i.e., https://your-jira-instance.atlassian.net.
+    - **Jira Connect App Password**.
 
-1. Set the Octopus Server URL in Octopus Deploy.
+    Ensure the **Is Enabled** property is set as well.
+
+3. In Octopus Deploy Configure the Release Note Options.
+
+    - **Jira username/password**: Set these values to allow Octopus to connect to Jira and retrieve Jira issue (work item) details when viewing packages or creating releases.
+
+    <!-- Lee: I noted work item in brackets above to make the connection between the two terms. Is that sufficent? I've used Jira issue primarily on this page as it's what the audience understands but work item is our term. Need advice on this. -->
+
+    These are optional and depend on the Jira Base URL. If they are not provided then just the raw work item references will be used as the work item link descriptions. If they are provided the work item's title will be used as the work item link's description.
+
+    The password should be an API Token, rather than an actual password. You can create an API token from an Atlassian account in the 'Security' area.
+    
+    - **Release Note Prefix**: Set this if you would like Octopus to scan the Jira issue's comments for release notes. This allows you to set a specific release note for a work item that is different to its title.
+
+    This is an optional setting and depends on the Jira Username and Jira Password being set. If specified, Octopus will look for a comment that starts with the given prefix text and use whatever text appears after the prefix as the release note, which will come through to the release notes templates etc as the work item link's description. If not comment is found with the prefix then Octopus will default back to using the title for that work item.
+
+    For example, a prefix of `Release note:` can be used to identify a customer friendly issue title vs a technical feature or bug fix title.
+
+1. Ensure the Octopus Server URL in set in Octopus Deploy.
 
     Navigate to the **{{Configuration,Nodes}}** page and ensure you have set the Server URI field to your Octopus Server's base URL. i.e., https://my-company.octopus.app/ or https://my-company-internal-name/
+    Note: Octopus passes this value to Jira so it can build hyperlinks back to the deployments from its UI. It never actually tries to connect to this Url itself.
```

Comment:
```suggestion
    Note: Octopus passes this value to Jira so it can build hyperlinks back to the deployments from its UI. It never actually tries to connect to this Url itself. If you are using Octopus Cloud, this value will be automatically set.
```